src/main/java/net/snowflake/ingest/streaming/internal/FileColumnProperties.java

```diff
@@ -206,6 +223,7 @@ void setMaxStrNonCollated(String maxStrNonCollated) {
   public String toString() {
     final StringBuilder sb = new StringBuilder("{");
     sb.append("\"columnOrdinal\": ").append(columnOrdinal);
+    sb.append(", \"fieldId\": ").append(fieldId);
```

Comment:
won't this cause immediate IT failures? the server doesn't expect fieldId in the json, so the deserializer is going to barf?

Reply:
I think the `toString` method is used for logging. The deserializer follows the `@JsonProperty` tag.

Reply:
I see, that's odd coz we're taking the pain to emit proper json here!